### PR TITLE
feat: add the demos for how to config the params in FS.

### PR DESCRIPTION
**Pull request by cqqqq777**
#### What type of PR is this?
Add example

#### What this PR does / why we need it (English/Chinese):

add demos for how to config the params in [FS](https://github.com/cloudwego/hertz/blob/develop/pkg/app/fs.go#L105).
#### Which issue(s) this PR fixes:

[this issue](https://github.com/cloudwego/hertz/issues/699)

### CODE REVIEW

File: file/staticFile/main.go [hunk 1]
```diff
@@ -17,13 +17,26 @@
...
 )
 
 func main() {
 	h := server.Default(server.WithHostPorts("127.0.0.1:8080"))
 
+	h.Static("/static", "./")
+
 	h.StaticFile("/main", "./main.go")
 
+	h.StaticFS("/static1", &app.FS{
```
Comment (li-jin-gou) [inline file/staticFile/main.go:34]: image


It seems that only two parameters are used

**Review by li-jin-gou — CHANGES_REQUESTED** [review]

**Comment by cqqqq777** [comment]
Now,I've added all params, please check it

**Comment by cqqqq777** [comment]
I can execute and pass `go vet -stdmethods=false $(go list ./...)`  and `test -z "$(gofumpt -l -extra .)"` in localhost

**Comment by welkeyever** [comment]
run this: `gofumpt -l -extra .`

**Review by li-jin-gou — CHANGES_REQUESTED** [review]
Why are there so many code changes?

File: file/staticFile/main.go [hunk 2]
```diff
@@ -17,13 +17,39 @@
...
 
+	h.Static("/static", "./")
+
 	h.StaticFile("/main", "./main.go")
 
+	indexNames := []string{"1.txt", "2.txt"}
+	h.StaticFS("/static1", &app.FS{
+		Root:        "./",
+		PathRewrite: app.NewPathSlashesStripper(1),
+		PathNotFound: func(_ context.Context, ctx *app.RequestContext) {
```
Comment (li-jin-gou) [inline file/staticFile/main.go:39]: Need to add some comments to help users understand

**Review by li-jin-gou — CHANGES_REQUESTED** [review]

File: file/staticFile/main.go (hunk 2 above)
Comment (welkeyever) [reply file/staticFile/main.go:39]: +1, maybe add them to readme.
Explain: what & how for this fields.

Comment (cqqqq777) [reply file/staticFile/main.go:39]: ok

Comment (li-jin-gou) [reply file/staticFile/main.go:39]: image
No classification and title introduce

**Review by li-jin-gou — APPROVED** [review]
LGTM

**Pull request merged**